tor/strings/en_US.yml

```diff
@@ -259,6 +259,19 @@ formatting_issues:
 
     \    int x = 0;\n
     \   int y = x * 100;"
+  unescaped_heading: "**Unescaped heading syntax**: It looks like you have an unescaped heading in your transcription, this may cause it to render as a heading instead of a raw hashtag. For example:
```

Comment:
On second thought I think we should alter this again to make it a bit clearer.

Something like "It looks like you intended to transcribe a hashtag, but made a heading (i.e. "big text") instead. This is caused by not escaping the number sign (`#`) in front of the hashtag. For example: [...]"

I should have noticed that sooner, sorry.

Reply:
No problem